server.js

```diff
@@ -80,7 +80,7 @@ server.post("/api/login", (req, res) => {
 server.get("/api/users", restricted, (req, res) => {
   Users.find()
     .then(users => {
-      res.json(users);
+      res.status(200).json(users);
```

Comment:
This was fine since the default is 200; what should certainly send an explicit status is the `res.send(err)` below so that it doesn't default to 200!